Task1/Program.cs

```diff
@@ -10,7 +10,262 @@ class Program
     {
         static void Main(string[] args)
         {
+            //HourPassed(7400);
+            //PlaceOfVariable(1, 4, 3);
+            //IsOdd(135);
+            //Calculation();
+            //Calculator();
+            //RangeOfNumber();
+            //Translator();
+            //Premium();
+            AditionalCalculator();
+            Console.ReadKey();
+        }
+
+        public static void HourPassed(int seconds)
+        {
+            int hours = seconds / 3600;
+            Console.WriteLine($" in {seconds} seconds is {hours} hours");
+        }
+
+        public static void PlaceOfVariable(int a, int b, int c)
+        {
+            if (a < b && b < c)
+            {
+                Console.WriteLine("B is between A and C");
+            }
+            else
+                Console.WriteLine("B is not between A and C");
+        }
+
+        public static void IsOdd(int number)
+        {
+            int amountOfRank = Convert.ToString(number).Length;
+            if (number % 2 == 1 && amountOfRank == 3)
+            {
```

Comment:
No need to use operator brackets since you've already skipped them in else statement.